monero-outreach-docs/en/quickfacts/Quickfacts.md

```diff
@@ -122,7 +120,7 @@ Although Monero is already available and being used across the globe, the commun
 + **_[Mastering Monero](https://masteringmonero.com/)_**
 + **_[Satis Group Report](https://research.bloomberg.com/pub/res/d37g1Q1hEhBkiRCu_ruMdMsbc0A)_**
 
-**Monero Quick Facts - Revised 2/1/2019** 
+**Monero Quick Facts - Revised 5/2/2019** 
```

Comment:
Just to confirm, should't it be 5/20 as in the technical fundamentals?

Reply:
Yes it should, all the other dates are 5/20.

Reply:
fixed

Reply:
The pdf and .sla file didn't have this error. It was the .md that had it. Thank you for the notice :).